mis_builder/models/kpimatrix.py

```diff
@@ -164,6 +164,8 @@ def __init__(self, env):
         self._sum_todo = {}
         # { account_id: account_name }
         self._account_names = {}
+        # id of the currency
+        self.currency_id = None
```

Comment:
I suggest naming this currency instead of currency_id and passing it in the KpiMatrix constructor.

Reply:
Should we have here in KpiMatrix class the id of the currency or the currency itself?
The value stored here is then passed to the Mis Report Style render method.